build.fsx

```diff
@@ -47,8 +47,17 @@ let gitHome = "https://github.com/" + gitOwner
 let gitName = "VisualFSharpPowerTools"
 let cloneUrl = "git@github.com:fsprojects/VisualFSharpPowerTools.git"
 
-// Ensure to use MSBuild 12 for the main solution
-setEnvironVar "MSBuild" (ProgramFilesX86 @@ @"\MSBuild\12.0\Bin\MSBuild.exe")
+// Ensure to use MSBuild 12 for the main solution if it exists, otherwise use version 14
+let msBuildVersionToTry = ["12.0";"14.0"]
```

Comment:
Thanks for fixing this. I think the logic could be simplified as follows:

Check if MsBuild 12 exists and set priority to MsBuild 12. Otherwise, do nothing.